src/Widgets/TestimonialSliderWidget/TestimonialSliderWidgetComponent.js

```diff
@@ -16,12 +16,18 @@ Scrivito.provideComponent('TestimonialSliderWidget', ({ widget }) => {
           testimonials.map(testimonial =>
             <div key={ testimonial.id() }>
               <h1 className="quote-headline text-center">&quot;</h1>
-              <p className="h4 text-center">
-                { testimonial.get('testimonial') }
-              </p>
-              <p className="small text-center">
-                { testimonial.get('author') }
-              </p>
+              <Scrivito.ContentTag
```

Comment:
* [ ] FYI not the focus of this PR, but to make this feature complete, the image should also be editable in-place

Reply:
That will be hard to do, since the Slick Slider component used already has a click event on the images - to move to the slider. I don't think it is worth it to build code, that "fights" with Slick-Slider who can use the click or not.